packages/cardpay-reward-programs/.vscode/settings.json

```diff
@@ -1,3 +1,12 @@
 {
-  "python.analysis.extraPaths": ["./streamlit"]
-}
+    "python.testing.pytestArgs": [
+        "tests",
+        "-rs",
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.analysis.extraPaths": [
+        "./streamlit",
+        "./__pypackages__/3.9/lib"
+    ]
```

Comment:
Cheers. Actually perhaps I should include this in all subpackages. So ppl wanting to look at single subpackage dont have to use python codeworkspace for this. Sorry if this impeded u